beacon-chain/operations/attestations/prepare_forkchoice.go

```diff
@@ -82,7 +83,11 @@ func (s *Service) batchForkChoiceAtts(ctx context.Context) error {
 // This aggregates a list of attestations using the aggregation algorithm defined in AggregateAttestations
 // and saves the attestations for fork choice.
 func (s *Service) aggregateAndSaveForkChoiceAtts(atts []*ethpb.Attestation) error {
-	aggregatedAtts, err := helpers.AggregateAttestations(atts)
+	clonedAtts := make([]*ethpb.Attestation, len(atts))
+	for i, a := range atts {
+		clonedAtts[i] = proto.Clone(a).(*ethpb.Attestation)
```

Comment:
Any chance we can use a custom copy? We have one in `beacon-chain/state/getters.go`